<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FutureTableTop</class>
 <widget class="QWidget" name="FutureTableTop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>80</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>选择标的</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="chooseSymbol">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>标的    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="symbol">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>最新价  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lastPrice">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>买一价 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bid1Price">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>买一量 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bid1Num">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>卖一价 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ask1Price">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>卖一量 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ask1Num">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>涨跌  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="zhangdie">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>最高价 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="highPrice">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>最低价 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lowPrice">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>昨结算价</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="preSettlePrice">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>更新时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="time">
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
